src/components/group-view.tsx

```diff
@@ -77,12 +77,14 @@ export class GroupViewComponent extends BaseComponent<IProps, {}> {
       );
     }
     else {
+      const readOnly = primaryDocument.type === "publication";
```

Comment:
I was hoping to keep document-specific actions in the `Document` model rather than having scattered logic like this, but I don't think it's worth making a structural change for this fix

Reply:
I debated having this check in the `DocumentComponent` but since in every other case the DocumentComponent is told whether it's readOnly or not, it seemed more consistent to make that determination here.